closed/autoconf/custom-hook.m4

```diff
@@ -49,10 +49,185 @@ AC_DEFUN_ONCE([CUSTOM_EARLY_HOOK],
   OPENJ9_CONFIGURE_WARNINGS
   OPENJ9_CONFIGURE_JITSERVER
   OPENJ9_CONFIGURE_INLINE_TYPES
+  OPENJ9_CONFIGURE_OPENJDK_THREAD_SUPPORT
   OPENJ9_THIRD_PARTY_REQUIREMENTS
   OPENJ9_CHECK_NASM_VERSION
 ])
 
+AC_DEFUN([CONFIGURE_OPENSSL],
```

Comment:
This is not in its proper location (after reordering).

Reply:
`CONFIGURE_OPENSSL` is in right order unless changing it to `OPENJ9_CONFIGURE_OPENSSL` for `AC_DEFUN` name consistency. Renamed it.